qiskit_nature/second_q/formats/fcidump/fcidump.py

```diff
@@ -41,6 +41,7 @@ class FCIDump:
 
     hij: np.ndarray
     """The alpha 1-electron integrals."""
+    # TODO: automatically handle to-chemist index order conversion?
```

Comment:
I am not sure what the FCIDump format dictates. Will look into this in more detail next week

Reply:
In fact, the FCIDump format dictates chemistry notation. Thus, I will handle the conversion directly here in order to be consistent :+1: